lxd/device/nic.go

```diff
@@ -36,6 +36,7 @@ func nicValidationRules(requiredFields []string, optionalFields []string) map[st
 		"vlan":                    shared.IsAny,
 		"hwaddr":                  networkValidMAC,
 		"host_name":               shared.IsAny,
+		"host_table":              shared.IsUint32,
```

Comment:
Any reason for using the single key vs having `ipv4.host_table` and `ipv6.host_table`?

Reply:
@stgraber not particularly. I suppose one could have different custom routing table IDs for IPv4 and IPv6 (although you can add IPv4 and IPv6 routes to the same routing table ID).

I'd be happy to split this into `ipv6.host_table` and `ipv4.host_table` if thats more useful?

Reply:
Yeah, I think we may as well do it now rather than have to split things up later.

I often have one routing table per ISP, so have two tables for IPv4 and one for IPv6 in my case. Not sure how common this is but I'm sure someone else has a similar setup :)

Reply:
@stgraber ok cool, will also apply to the ipvlan PR too.